test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -192,13 +189,22 @@ def reply_to_message_at_index(self, index: int, message: str):
         message_object_to_reply_to = self.get_message_at_index(index)
         verify(not is_null(message_object_to_reply_to), "Message to reply to is loaded")
         move_mouse_over_object(message_object_to_reply_to)
-        click_obj_by_name(ChatComponents.REPLY_TO_MESSAGE_BUTTON.value)
+        foundreplyToButton = getChildrenWithObjectName(message_object_to_reply_to, "replyToMessageButton")[0]
+        verify(not is_null(foundreplyToButton), "Reply button found")
+        move_mouse_over_object(foundreplyToButton)
+        time.sleep(1)
+        click_obj(foundreplyToButton)
         self.send_message(message)
     
     def edit_message_at_index(self, index: int, message: str):
         message_object_to_edit = wait_and_get_obj(ChatComponents.CHAT_LOG.value).itemAtIndex(int(index))
+        verify(not is_null(message_object_to_edit), "Message to edit is loaded")
         move_mouse_over_object(message_object_to_edit)
-        click_obj_by_name(ChatComponents.EDIT_MESSAGE_BUTTON.value)
+        foundEditButton = getChildrenWithObjectName(message_object_to_edit, "editMessageButton")[0]
```

Comment:
```suggestion
        found_edit_button = getChildrenWithObjectName(message_object_to_edit, "editMessageButton")[0]
```

Reply:
Same comment than before here, as well. It would be greate to have `editMessageButton` as a constant.

Reply:
Done